content/pages/meetup/2023/0726-nycu.rst

```diff
@@ -18,29 +18,46 @@ Agenda
 --------
 
 * 18:30-19:30 Free discussion
-* 19:40-20:10 Subject (sharing): Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis
-* 20:10-21:30 Subject (discussion): Preparations for August scisprint  
+* 19:40-20:10 Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis
+* 20:10-20:30 sciwork grand project (Yung-Yu)
+* 20:30-21:30 Preparational meeting for August scisprint  
 
 |
 
 Subjects
 ------------------
 
-* **Sharing: Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis**
+Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    Speaker : Nian-Cheng - A graduate student in NYCU Institute of Artificial Intelligence Innovation.
+Speaker : Nian-Cheng - A graduate student in NYCU Institute of Artificial Intelligence Innovation.
     
-    In this topic, I leveraged data from depth cameras for AI model data processing and introduces the essential tools I utilized for this purpose.
+In this topic, I leveraged data from depth cameras for AI model data processing and introduces the essential tools I utilized for this purpose.
 
-* **Discussion: Preparations for August scisprint**
+sciwork grand project
++++++++++++++++++++++
 
-    Additionally, we will discuss the following items for preparing August scisprint. 
+Host: Yung-Yu
 
-    * Task arrangement for volunteers
-    * Promotion plan for August scisprint
-    * Preparations for career conversation
+In the past years sciwork communit is using the portal site https://sciwork.dev/
```

Comment:
Text with proper line-breaking will be easier to maintain in the future.  A long line (like > 79 chars) will carry too much diff when being modified in the future.  Properly broken lines are also a must for a normal paragraph (which has sufficiently many sentences).